content/blog/rag-pipelines-from-scratch/index.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: blog-post
+title: "RAG Pipelines from scratch"
+description: "Let's build a simple RAG Pipeline with Haystack 2.0 by just connecting three components: a Retriever, a PromptBuilder and a Generator."
+featured_image: thumbnail.png
+images: ["blog/rag-pipelines-from-scratch/thumbnail.png"]
+alt_image: The Republic of Rose Island
+toc: True
+date: 2023-10-27
+last_updated:  2023-10-28
```

Comment:
You should update these two dates to appear at the top of the articles

Reply:
Which date should I set?